remoteit/desktop
# [DESK-392] update cli version and handle update change

### Description

Update cli version. Also handle the update change so that if services aren't running the whole process won't fail.

### Checklist




### Test plan



1. Uninstall and install v2.5.22 of desktop
2. Verify version of cli on command line
3. Install this new version
4. Check the version of cli on the command line again
5. CLI version should be 1.0.3
6. Desktop should still function normally

### Ticket



<https://remoteit.atlassian.net/browse/https://remoteit.atlassian.net/browse/DESK-392>
